tests/src/rules/no-anonymous-default-export.js

```diff
@@ -43,6 +44,7 @@ ruleTester.run('no-anonymous-default-export', rule, {
       test({ code: 'export default \'foo\'', errors: [{ message: 'Assign literal to a variable before exporting as module default' }] }),
       test({ code: 'export default `foo`', errors: [{ message: 'Assign literal to a variable before exporting as module default' }] }),
       test({ code: 'export default {}', errors: [{ message: 'Assign object to a variable before exporting as module default' }] }),
+      test({ code: 'export default foo(bar)', errors: [{ message: 'Assign call result to a variable before exporting as module default' }] }),
```

Comment:
this constitutes a breaking change if it's `false` by default; it'd need to default to `true` so it can be semver-minor

Reply:
Ahh, yes. Will fix that.